copyables/gencert.sh

```
@@ -1,20 +1,20 @@
 #!/bin/bash
 set -e
 
-/usr/bin/vpnserver start 2>&1 >/dev/null
+/usr/local/bin/vpnserver start 2>&1 >/dev/null
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)